spec/commands/inject_spec.rb

```diff
@@ -68,6 +68,13 @@
       str = "gem 'rack-obama', '> 0', :group => [:development]"
       expect(gemfile).to include str
     end
+
+    it "add gem with multiple group in gemfile" do
+      bundle "inject 'rack-obama' '>0' --group=development,test"
+      gemfile = bundled_app("Gemfile").read
+      str = "gem 'rack-obama', '> 0', :group => [:development, :test]"
```

Comment:
I edited this line : "add gem with multiple group in gemfile"  

and I think this line : `gem 'rack-obama', '> 0', :group => [:development, :test]` is correct. I followed this link : http://bundler.io/v1.3/groups.html  examples .